docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -93,7 +93,7 @@ Additional threat intelligence data is generated for indicator match alerts when
 
 Shows the number of individual threats matching the alert in reverse chronological order, with the most recent threats at the top. Each threat is labelled with values from the `matched.field` and `matched.atomic` fields. All mapped fields are displayed for each matched threat.
 
-NOTE: If the alert does not contain threat intelligence data, the *Threat Intel* tab displays a message that none is available and provides a link to Threat Intel module documentation.
+NOTE: If the alert does not contain threat intelligence data, the *Threat Intel* tab displays a message that none is available and provides a link to Threat Intel module documentation. The event enrichment query uses the indices specified in the `securitySolution:defaultThreatIndex` advanced setting. For more information, see <<update-threat-intel-indices, Update default Elastic Security threat intelligence indices>>.
```

Comment:
I thought that "event enrichment query" seemed like a new concept without much context or connection to the first sentence in the note. Also, my understanding of how this works is that the "query" is internal logic, so it may not be necessary to use such a technical description. Here's a possible revision. 
```suggestion
NOTE: If the alert does not contain threat intelligence data, the *Threat Intel* tab displays a message that none is available and provides a link to Threat Intel module documentation. If the alert does contain threat intelligence data, it is enriched using the indices specified in the `securitySolution:defaultThreatIndex` advanced setting. For more information, see <<update-threat-intel-indices, Update default Elastic Security threat intelligence indices>>.
```